requests_ntlm/requests_ntlm.py

```diff
@@ -13,25 +13,28 @@ class HttpNtlmAuth(AuthBase):
     Supports pass-the-hash.
     """
 
-    def __init__(self, username, password, session=None):
+    def __init__(self, username, password, session=None, domain=None):
         """Create an authentication handler for NTLM over HTTP.
 
-        :param str username: Username in 'domain\\username' format
+        :param str username: Username
```

Comment:
Rather than remove the old note, let's just extend it to explain what's going on.